kubectl-fdb/cmd/cordon_test.go

```diff
@@ -21,121 +21,219 @@
+					instancesToRemove = append(instancesToRemove, resCluster.Spec.ProcessGroupsToRemove...)
+					instancesToRemoveWithoutExclusion = append(instancesToRemoveWithoutExclusion, resCluster.Spec.ProcessGroupsToRemoveWithoutExclusion...)
+				}
 
 				Expect(err).NotTo(HaveOccurred())
 				// Use equality.Semantic.DeepEqual here since the Equal check of gomega is to strict
-				Expect(equality.Semantic.DeepEqual(input.ExpectedInstancesToRemove, resCluster.Spec.ProcessGroupsToRemove)).To(BeTrue())
-				Expect(equality.Semantic.DeepEqual(input.ExpectedInstancesToRemoveWithoutExclusion, resCluster.Spec.ProcessGroupsToRemoveWithoutExclusion)).To(BeTrue())
+				fmt.Printf("\ninstancesToRemove: %v\n", instancesToRemove)
+				Expect(equality.Semantic.DeepEqual(input.ExpectedInstancesToRemove, instancesToRemove)).To(BeTrue())
+				fmt.Printf("\ninstancesToRemoveWithoutExclusion: %v\n", instancesToRemoveWithoutExclusion)
+				Expect(equality.Semantic.DeepEqual(input.ExpectedInstancesToRemoveWithoutExclusion, instancesToRemoveWithoutExclusion)).To(BeTrue())
 			},
 			Entry("Cordon node with exclusion",
 				testCase{
 					nodes:                     []string{"node-1"},
 					WithExclusion:             true,
-					ExpectedInstancesToRemove: []string{"instance-1"},
+					ExpectedInstancesToRemove: []string{fmt.Sprintf("%s-instance-1", clusterName)},
 					ExpectedInstancesToRemoveWithoutExclusion: []string{},
+					clusterName:  clusterName,
+					clusterLabel: "",
 				}),
 			Entry("Cordon node without exclusion",
 				testCase{
 					nodes:                     []string{"node-1"},
 					WithExclusion:             false,
 					ExpectedInstancesToRemove: []string{},
-					ExpectedInstancesToRemoveWithoutExclusion: []string{"instance-1"},
+					ExpectedInstancesToRemoveWithoutExclusion: []string{fmt.Sprintf("%s-instance-1", clusterName)},
+					clusterName:  clusterName,
+					clusterLabel: "",
 				}),
 			Entry("Cordon no nodes with exclusion",
 				testCase{
 					nodes:                     []string{""},
 					WithExclusion:             true,
 					ExpectedInstancesToRemove: []string{},
 					ExpectedInstancesToRemoveWithoutExclusion: []string{},
+					clusterName:  clusterName,
+					clusterLabel: "",
 				}),
 			Entry("Cordon no node nodes without exclusion",
 				testCase{
 					nodes:                     []string{""},
 					WithExclusion:             false,
 					ExpectedInstancesToRemove: []string{},
 					ExpectedInstancesToRemoveWithoutExclusion: []string{},
+					clusterName:  clusterName,
+					clusterLabel: "",
 				}),
 			Entry("Cordon all nodes with exclusion",
+				testCase{
+					nodes:         []string{"node-1", "node-2"},
+					WithExclusion: true,
+					ExpectedInstancesToRemove: []string{
+						fmt.Sprintf("%s-instance-1", clusterName),
+						fmt.Sprintf("%s-instance-2", clusterName),
+					},
+					ExpectedInstancesToRemoveWithoutExclusion: []string{},
+					clusterName:  clusterName,
+					clusterLabel: fdbv1beta2.FDBClusterLabel,
```

Comment:
Is there a reason we add the cluster label here too but in other test cases we leave it empty?